### ISSUE TITLE
[Request] Make scrolling really long lists (e.g. Select) more performant

### ISSUE THREAD

**Issue by wc-matteo** [quasarframework/quasar#1599]
I've tried to use `Select` with 1000s of elements and it was not pretty (just loading the control was really really slow). Maybe integrating something like [this](https://clusterize.js.org/) would help.

**Comment by Frondor** [comment]
Not even twitter feed works well with 1000 records. It's not Quasar's fault, but probably a bad user implementation, see https://hackernoon.com/guys-were-doing-pagination-wrong-f6c18a91b232
As you propose another project dependency, I don't think the framework should care about these specific cases where developers can easily tackle it by implementing that behavior themselves.

You can always provide a PR every time you notice some bug, or like in this case, something is not performing well enough for you.

**Comment by wc-matteo** [comment]
I understand.

Just wondering, would the PR be accepted in this case though or, as you say, this isn't something the framework should accommodate for?

**Comment by Frondor** [comment]
I don't think so. I'm not a maintainer though, so don't take my word on this.
But I'm pretty sure Razvan won't implement such thing, which implies importing another dependency to the project, when this is something that should be addressed by the developer itself, since the case is very specific (most people implement this by themselves, or use a proper pagination approach), as I already said before.
I encouraged you to provide pull requests because I noticed you're requesting things once in a while, which isn't bad at all, but feels even better when you see people getting involved.
